openassessment/xblock/openassessmentblock.py

```diff
@@ -1127,6 +1133,13 @@ def is_closed(self, step=None, course_staff=None):
         if course_staff:
             return False, None, DISTANT_PAST, DISTANT_FUTURE
 
+        if self.date_config_type == 'course_end':
+            course = self.get_course()
+            if course.start and course.end:
```

Comment:
is it possible for these to not be truthy? what is this checking for?

Reply:
Locally, when the course just created. Start or end date doesn't get define. This is for the safety check.